editor/test/state.js

```diff
@@ -386,31 +441,32 @@ describe( 'state', () => {
 			expect( state ).to.eql( { uid: 'ribs', typing: true, focus: { editable: 'citation' } } );
 		} );
 
-		it( 'should insert after the specified block uid', () => {
-			const original = blocks( undefined, {
+		it( 'should replace the selected block', () => {
+			const original = deepFreeze( { uid: 'chicken', typing: false, focus: { editable: 'citation' } } );
```

Comment:
I didn't catch this before it was merged, but this really should have been split out into separate lines.